locale/en/get-involved/node-meetups.md

```diff
@@ -553,11 +553,11 @@ REQUIREMENTS
 
 ##### Kiev
 
-- [Meetup](https://www.meetup.com/KievNodeJS/)
-- Frequency of meetups - monthly
-- How to submit a talk?
+- [Meetup](https://www.meetup.com/NodeUA/), [Old group](https://www.meetup.com/KievNodeJS/)
+- Frequency of meetups - 1-8 times a month
```

Comment:
Up to 8 times a month?! Wow! 👍

Reply:
@fhemberger including lectures, workshops, discussions, with @aqrln @lundibundi @nechaido @belochub @DzyubSpirit and many others, some videos are here:

- [NodeUA](https://www.youtube.com/playlist?list=PLHhi8ymDMrQZmXEqIIlq2S9-Ibh9b_-rQ)
- [HowProgrammingWorks and JavaScript](https://www.youtube.com/playlist?list=PLHhi8ymDMrQZad6JDh6HRzY1Wz5WB34w0)
- [NodeUA Summer School](https://www.youtube.com/playlist?list=PLXzsasNqYIa2igoviH9LLT3Q7aymHWkpN)
- [KievNodeJs and Metarhia meetups](https://www.youtube.com/playlist?list=PLXzsasNqYIa0Cb2Z_dyaXJXf6ekbjD-Iu)
- [Asynchronous programming with Js and NodeJS lectures](https://www.youtube.com/playlist?list=PLHhi8ymDMrQZ0MpTsmi54OkjTbo0cjU1T)

Node.js and JavaScript meetups:
- [NodeUA (2088 developers, 41 events)](https://www.meetup.com/NodeUA/)
- [KievNodeJS (1380 developers, past 19 events)](https://www.meetup.com/KievNodeJS/)
- [HowProgrammingWorks (2040 developers, past 148 events)](https://www.meetup.com/HowProgrammingWorks/)

About 600 students each year attend lectures of specialty "Software Engineering" based on Node.js in National Technical University of Ukraine.